AlphaWallet/Market/Coordinators/UniversalLinkCoordinator.swift

```diff
@@ -192,7 +202,25 @@ class UniversalLinkCoordinator: Coordinator {
             guard let recoverAddress = Address(string: ethereumAddress.address) else { return false }
             let contractAsAddress = Address(string: signedOrder.order.contractAddress)!
             if signedOrder.order.nativeCurrencyDrop {
-                self.makeTokenHolder([""], signedOrder.order.contractAddress)
+                let amt: Decimal
+                if let amount = Decimal(string: signedOrder.order.count.description) {
```

Comment:
You also simplify this:

```
if let amount = Decimal(string: signedOrder.order.count.description) {
```

with:

```
if let amount = Decimal(exactly: signedOrder.order.count)
```

Looks a bit odd though. Do you really mean to read the value of `count`?

Reply:
Yes, count records the amount in szabo